### PR TITLE
fix(nav): make doc and bug buttons customizable

### PR DESCRIPTION
**Pull request by villebro**
### SUMMARY
Currently the documentation text and icon config parameters are not being respected in the `RightMenu` component. See the parameters here: https://github.com/apache/superset/blob/c0aeb2a75a06694402be0f15d1eb64631d3b9ef9/superset/config.py#L1307-L1310

This fixes that + adds similar overrides for the bug links. In addition, the old right menu FAB template is no longer being used, and can hence be deleted.

### AFTER
Here's the docs and buglinks with the new customizations (notice the top right hand corner next to "Settings"):
image

### BEFORE
Previously the `DOCUMENTATION_ICON` and `DOCUMENTATION_ICON` params weren't being used, and the bug link was not customizable:
image

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/views/components/RightMenu.tsx [hunk 1]
```diff
@@ -539,25 +539,38 @@ const RightMenu = ({
...
-        >
-          <i className="fa fa-question" />
-          &nbsp;
-        </StyledAnchor>
+        <>
+          <StyledAnchor
+            href={navbarRight.documentation_url}
+            target="_blank"
+            rel="noreferrer"
+            title={navbarRight.documentation_text || t('Documentation')}
```
Comment (kgabryje) [inline superset-frontend/src/views/components/RightMenu.tsx:547]: 1. Should we translate `documentation_text`? If not, we probably don't want to translate `bug_report_text` in line 564?
2. Are you sure that `t()` will work correctly with dynamic expression like here (text depends on result of `||` operator). Shouldn't we use `t('%s', navbarRight.documentation_text || 'Documentation')` instead? That's actually more of a question than suggestion, I'm not sure which style is preferred here

Comment (villebro) [reply superset-frontend/src/views/components/RightMenu.tsx:547]: Oof, the `bug_report_text` translation was a complete mistake from me! Will fix

Comment (villebro) [reply superset-frontend/src/views/components/RightMenu.tsx:547]: Regarding question 2, I think the pattern in `documentation_text` is correct, i.e. we want to use the string from the config as-is, and only fall back to the translated text if it's undefined.

**Review by kgabryje — APPROVED** [review]
LGTM!

**Pull request merged**